src/main/assembly/dist/bin/help.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Continuing my previous remark (I was called away :), earlier I though there was a conflict between the following two requirements:
* The working directory should stay the same. This would *not* be the case when using a strategy where you just `pushd` to the home-directory and `popd` afterwards. Not having this, however, is confusing when the command line application may take a file name (and therefore relative path) as an argument. Not all applications do, but it is sufficiently common to make this a general principle.
* The main jar can find its library dependencies. Currently this is achieved through relative paths in the main jar's `MANIFEST.MF` file.

It seems I was wrong about them being conflicting. The MANIFEST.MF file seems to resolve the paths to the libraries relative to the jar it is contained in, *not* relative to the working directory (which I previously thought).